Makefile

```diff
@@ -17,6 +17,10 @@ FAST_TESTS?=FALSE
 FAST_BENCH?=FALSE
 BENCH_OP_FLAVOR?=DEFAULT
 NODE_VERSION=20
+TFHE_CURRENT_VERSION:=$(shell grep '^version[[:space:]]*=' tfhe/Cargo.toml | cut -d '=' -f 2 | xargs)
```

Comment:
This could give the wrong result if a crate dependency is defined before the package version and its version is in new line 

That's unlikely

Reply:
Not sure if it exist, but inserting a utility which asserts that a stream is only one line and forwards it would be more secure (even if not perfect)

Reply:
though it looks like the Cargo.toml format does not allow new lines for { version = "..." } stuff

Reply:
I think you can have 
```
[dependencies.dep_name]
version = "1.0"
```

Reply:
yeah :|

Reply:
I guess I could trim the file once we find the word "dependencies"

Reply:
something like this maybe https://stackoverflow.com/questions/7103531/how-to-get-the-part-of-a-file-after-the-first-line-that-matches-a-regular-expres

Reply:
or maybe a light toml parser

Reply:
That seems overkill to me

Reply:
got something a bit more robust